### PR TITLE
[docs] [Integrations] Add a page for Akka Persistence Cassandra

### LINKED ISSUE #14712: [DocDB] Akka-Persistence-Cassandra integration with Yugabyte-DB
### Description

We need to add Akka to the integrations section of our documentation, the content for the same is written in this [doc](https://docs.google.com/document/d/1m02giuNeRa4XOjGmZM5Ed3sdq_6hgKxVedlAYuDDhyY/edit).

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #14712

### CODE REVIEW

**Review by ashetkar — APPROVED** [review]

**Review by HarshDaryani896 — APPROVED** [review]

File: docs/content/preview/integrations/_index.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ For information on YugabyteDB-compatible drivers and ORMs, refer to [Drivers and
 
 <ul class="nav yb-pills">
 
+  <li>
+    <a href="apache-kafka/">
```
Comment (HarshDaryani896) [inline docs/content/preview/integrations/_index.md:25]: Shouldn't this be `<a href="akka/">`

File: docs/content/preview/integrations/akka.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    Press enter when you're prompted to name the application.
+
+1. Add the following code in `build.sbt` file from your project's home directory.
+
+    ```sh
+    cd scala-3-project-template
+    ```
+
+    ```scala
+
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka.md:51]: extra space

File: docs/content/preview/integrations/akka.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+Akka Persistence enables stateful akka-actors to persist their state so that it can be recovered when an actor is either restarted, such as after a JVM crash by a supervisor or a manual stop-start, or migrated in a cluster.
+The Akka Persistence Cassandra plugin allows you to use Apache Cassandra as a backend for Akka Persistence and Akka Persistence Query. It uses Alpakka Cassandra for Cassandra access which is based on the Datastax Java Driver.
+Alpakka Cassandra offers an Akka Streams API on top of a `CqlSession` from the Datastax Java Driver version 4.0 and later.
+
+The following section illustrates how to run an Akka Persistence Cassandra based application with YugabyteDB.
+
+## Prerequisites
+
+To use Akka Persistence Cassandra plugin, ensure that you have the following:
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka.md:26]:
```suggestion
To use the Akka Persistence Cassandra plugin, ensure that you have the following:
```

File: docs/content/preview/integrations/akka.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+
+```cql
+select * from akka.messages
+```
+
+The output should include an event with a `persistance_id` same as the `account_id` (b2b51f94-b021-4fb8-ba33-219cd0aea3c1) obtained after [running the application](#run-the-application).
+
+## Limitations
+
+Some older versions of Akka Persistence Cassandra use Cassandra Materialized Views which may not work with YugabyteDB. So, you need to use versions of Akka Persistence Cassandra that *do not* use Cassandra Materialized Views.
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka.md:271]:
```suggestion
Some older versions of Akka Persistence Cassandra use Cassandra Materialized Views which may not work with YugabyteDB. Use versions of Akka Persistence Cassandra that *do not* use Cassandra Materialized Views.
```

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/integrations/akka.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/akka.md:51]: nice catch . Thanks Harsh.

File: docs/content/preview/integrations/_index.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/_index.md:25]: nice catch. Thanks Harsh.

**Pull request merged**